doc/data/messages/d/disallowed-name/good.py

```diff
@@ -1,2 +1,2 @@
-def something_meaningful():
-    pass
+def print_fruit():
+    print("apples")
```

Comment:
We should add the print in Bad.py too if we change this.

Reply:
it's already there :)
```
def foo():  # [disallowed-name]
    print("apples")

```